src/Components/SearchBarDetails.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+/**
+ * Representing the details from the searched character
+ */
+const SearchBarDetails = ({ searchedItemDetails, showSearchBarDetails }) => {
+
+    var divStyle = {
+        fontSize: '18px',
+        padding:0
+        };
```

Comment:
Create a separate JS file with common themes and put that there, you will eventually need more of those in a central place, and not in components as hardcoded values.

Reply:
Yep, I was planning to do that in the future.